package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "ship-components-upload",
+  "version": "0.1.0",
+  "description": "Upload button component",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "grunt server",
+    "build": "grunt build",
+    "test": "./node_modules/jest-cli/bin/jest.js src/js --coverage",
+    "coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ship-components/ship-components-upload.git"
+  },
+  "keywords": [
+    "react",
+    "button",
+    "upload"
+  ],
+  "author": "Sepand Assadi <sepand.assadi@sony.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ship-components/ship-components-upload/issues"
+  },
+  "homepage": "https://github.com/ship-components/ship-components-upload#readme",
+  "devDependencies": {
+    "attr-accept": "^1.0.3",
+    "autoprefixer": "^6.1.0",
+    "babel": "^6.0.15",
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.1.4",
+    "babel-eslint": "^4.1.5",
+    "babel-jest": "git+https://github.com/isuttell/babel-jest.git#master",
```

Comment:
I forked this a while back to fix a issue. I believe it's been fixed. We should switch this back to the original package.